test_vm/vagrant_config_defaults.rb

```diff
@@ -2,6 +2,6 @@
   {nfs: true, nfs_version: 4, mount_options: ["noatime", "nodiratime"]}
 
 # Default options
-CONFIG['memory'] = "1100"
+CONFIG['memory'] = "2048"
```

Comment:
Reduced to `1500` and tests including Oracle+DB2+Informix are still passing. I'll reduce further and see where things break.